skimage/io/_io.py

```diff
@@ -130,6 +130,9 @@ def imsave(fname, arr, plugin=None, **plugin_args):
             plugin = 'tifffile'
     if is_low_contrast(arr):
         warn('%s is a low contrast image' % fname)
+    if (type(arr) == np.ndarray and arr.dtype == bool) or \
+        any(isinstance(ele, bool) for ele in arr):
```

Comment:
@soupault you're right. A test:
```
In : boolist = [[True, False, True], [False, True, False], [True, False, False]]
In : any(isinstance(elem, bool) for elem in boolist)
Out: False
In : any(isinstance(elem, bool) for elem in boolist[0])
Out: True
```
Maybe something like this could be used:
```
if type(input_array) is not np.ndarray:
    input_array = np.asarray(input_array)
```